include/http/middleware/system.js

```diff
@@ -20,6 +20,7 @@ module.exports = pb => ({
     parseUrl: async (ctx, next) => {
         ctx.app.requestsServed ? ctx.app.requestsServed++ : 1;
         ctx.req.hostname = ctx.req.headers.host;
+        ctx.req.urlObj = url.parse(ctx.req.url, true);
```

Comment:
In hind sight, we should have (I say we.  ehm.  I should have) done this from the start and not modified the default req.url property.  It caused a lot of headache in the CMS proper.